### ISSUE TITLE
Guide to admin login

### ISSUE THREAD

**Issue by jtagcat** [drakkan/sftpgo#485]
`GET /` directs to `/web/client/login`

Admin credentials don't work there, they work on `/web/admin/login`. Using admin credentials on the WebClient gives `invalid credentials`. It's confusing as hell, especially after first setup.

Do offer a link on the login pages to switch between admin and web client login. Make them distinct.

**Comment by drakkan** [comment]
Hi,

it is confusing, you are right. I think adding a link to the web admin on the web client login page  is not a good idea. For example some SFTPGo users setup a reverse proxy and external web clients cannot reach the admin interface at all.

I could change the default redirect to something like `/web` and on that page add the links to admin and client login pages

**Comment by jtagcat** [comment]
What's the difference in connectability?

My proposed:
```
/  ──►  /web/client/login ──► /web/client
                      │
                      │
                   'Admin login' button
                      │
                      └── ───►  /web/admin/login
                                 │
                                 │

                                 ───► can't connect is ok
```

Choose your login:
```
            ┌───► /web/client/login
            │
            │
/ ────► choose
            │
            │
            └───► /web/admin/login
                        │
                        │
                        ▼
                  can't connect is ok
```

**Comment by jtagcat** [comment]
Having a simple 'switch to admin/webclient' button on both logins would do it fine. It doesn't matter if the user can connect to it.

**Comment by drakkan** [comment]
Hi,

you can give to your external users only the web client login link and they cannot reach the web admin.

You setup a reverse proxy and so from Internet you can only reach `/web/client` and its subdirs. There is no need for disambiguation for external users in this case. If I add a link to the web client interface I'm sure someone will ask for an option to disable it.

Internal users can connect to both web client and web admin. 

If I add a link on the login pages I also have to add a per-binding option to allow to disable it. I'll do a more accurate evaluation the next week

**Comment by drakkan** [comment]
A link on the login pages is more user friendly than a "Choose your login" page, thank you for your suggestion. I also added a new setting to hide the login link on one or both login pages

**Comment by alfonsrv** [comment]
My god that was probably the most frustrating and confusing part about this container. Re-created it like 5 times thinking the database might *somehow* be corrupted.

**Comment by drakkan** [comment]
> My god that was probably the most frustrating and confusing part about this container. Re-created it like 5 times thinking the database might _somehow_ be corrupted.

This is fixed in the edge based versions, and the fix will be included in v2.2.x
